.lando.yml

```diff
@@ -8,6 +8,8 @@ services:
   appserver:
     xdebug: false
     build_as_root:
+      - bash /app/.lando/scripts/configure-syslog.sh
+      - bash /app/.lando/scripts/configure-xhprof.sh
```

Comment:
IMHO, xhprof should be enabled on demand,

Reply:
i appreciate the contribution! i think we can get this merged, but a couple of thoughts:

- i agree with @sharique, i don't think every person running lando with blt will want xhprof, so my preference is to have a command that enables / disables it (like we have for xdebug). 
- is it possible to add something to tooling to start / stop the process? otherwise my preference would be an additional recipe command that would add this new stuff into the setup but not have it there by default.

Reply:
For xdebug we have it available in recipe, I'm not sure if we can make it configurable just through .lando.yml 

Also XHPROF is not enabled by default, we expect the XHPROF module to be enabled in the site to load it

Lastly, I would love to push for everyone to have it so they at-least know there is something like it available for use (I still don't expect all devs to even open this file but at-least if they do they will read about it here and will have one excuse less to not use such tools during development).